src/hooks/use-suggestions.ts

```diff
@@ -30,6 +30,7 @@ export enum Logical {
     OR = 'OR'
 }
 
+// exported only for createColorSchema
```

Comment:
The comment is not needed, please remove

Reply:
done